server/src/librelay/relay_server.go

```diff
@@ -331,8 +339,35 @@ func (relay *relayServer) awaitTransactionMined(tx *types.Transaction) (err erro
 	return nil
 }
 
-func (relay *relayServer) UnregisterRelay() error {
-	return relay.Unstake()
+func (relay *relayServer) RemoveRelay(ownerKey *ecdsa.PrivateKey) (err error) {
```

Comment:
its relay owner function to call, and it calls directly RelayHub - it doesn't call the relay at all.